financial-templates-lib/price-feed/CreatePriceFeed.js

```diff
@@ -205,7 +206,8 @@ const defaultConfigs = {
       { type: "cryptowatch", exchange: "bitstamp" }
     ]
   },
-  COMPUSD: {
+  "COMP/USD": {
+    // Kovan uses the "/"
```

Comment:
at least we are consistently inconsistent 👀

Reply:
we can't go wrong!